apps/hash-graph/lib/type-fetcher/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "type-fetcher"
+version = "0.0.0"
+edition = "2021"
+publish = false
+description = "RPC service definition to fetch external BP types"
+
+[dependencies]
```

Comment:
I'm quite happy that we don't have to keep the dependency versions in sync manually anymore